docs: create ALB with Gateway API instead of Ingress (#937)
kgateway-dev/kgateway.dev: content/docs/envoy/2.1.x/integrations/aws-elb/alb.md

## Step 2: Create an ALB with the AWS Load Balancer controller

> [!NOTE]
> Creating an ALB with {{< reuse "kgw-docs/snippets/k8s-gateway-api-name.md" >}} resources requires AWS Load Balancer Controller version 2.14.0 or later. Earlier versions can create an ALB only through an Ingress resource.

{{< reuse "kgw-docs/snippets/aws-elb-controller-install.md" >}}

> 5. Use another Gateway resource to define your ALB. When you apply these resources, the AWS Load Balancer Controller creates the ALB in your account.

Reviewer: ```suggestion
5. Create the Gateway API resources that define your ALB. When you apply these resources, the AWS Load Balancer Controller creates the ALB in your account.
```
